activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/Direction.scala

```diff
@@ -0,0 +1,18 @@
+package zio.flow.activities.twilio
+
+import zio.flow.Remote
+import zio.schema.DeriveSchema
+
+sealed trait Direction
+object Direction {
+  case object inbound          extends Direction
+  case object `outbound-api`   extends Direction
+  case object `outbound-call`  extends Direction
+  case object `outbound-reply` extends Direction
+
+  implicit val schema = DeriveSchema.gen[Direction]
+
+  object Accessors {
```

Comment:
Because of not being able to follow standard naming conventions, we also cannot have accessors with the same name but lowercase, as they collide with the object names